invest/templates/invest/high-potential-opportunity-detail.html

```diff
@@ -0,0 +1,231 @@
+{% endblock %}
+
+{% block head_title %}
+    High Potential Opportunities - great.gov.uk
+{% endblock %}
+
+{% block content %}
+<section id="hero" style="background-image: url({{ page.hero_image.url }})">
+    <div class="container">
+        <p class="breadcrumbs body-text">
+            <span class="label verbose">You are here:</span>
+            <a href="/">Home</a>
+            <span class="separator">,</span>
+            <span>Industry news</a>
+            <span class="separator">,</span>
+            <span class="current">{{ page.breadcrumbs_label }}</span>
+        </p>
+
+        <div class="grid-row">
+            <div class="column-two-thirds">
+                <h1 class="heading-xlarge">{{ page.heading }}</h1>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="contact-section">
+    <div class="container">
+        <div class="grid-row">
+            <div class="proposition-container">
+                {{ page.contact_proposition|add_export_elements_classes }}
+            </div>
+            <div class="button-container">
+                <a href="{% url 'uk-export-finance-pardot-funnel-start' %}" class="button">{{ page.contact_button }}</a>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="proposition-one">
+    <div class="container">
+        <div class="grid-row">
+            <div class="column-one-half">
+                {{ page.proposition_one|add_export_elements_classes }}
+            </div>
+            <div class="column-one-half">
+                {% if page.proposition_one_video %}
+                    {{ page.proposition_one_video }}
+                {% else %}
+                    <img src="{{ page.proposition_one_image.url }}" />
+                {% endif %}
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="opportunity-list">
+    <div class="container">
+        <div class="grid-row">
+            <h2 class="heading-large">{{ page.opportunity_list_title }}</h2>
+            <div class="column-one-third">
+                <ol class="list list-number">
+                    <li>{{ page.opportunity_list_item_one|add_export_elements_classes }}</li>
+                    <li>{{ page.opportunity_list_item_two|add_export_elements_classes }}</li> 
+                    <li>{{ page.opportunity_list_item_three|add_export_elements_classes }}</li> 
+                </ol>
+            </div>
+            <div class="column-two-thirds">
+                <img src="{{ page.opportunity_list_image.url }}" />
+
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="proposition-two">
+    <div class="container">
+        <div class="grid-row">
+            <div class="column-one-half">
+                <div class="proposition-two-container">
+                    {{ page.proposition_two|add_export_elements_classes }}
+                </div>
+                <ul class="top-level-list">
```

Comment:
elements has the classes `panel-vertical-narrow` and `panel-vertical-wide` that add a border to the top in blue. you could use that and override the colour here

Reply:
### panel-vertical-narrow gives us
![image](https://user-images.githubusercontent.com/5485798/45371189-e3dd4b80-b5e1-11e8-9efa-8b8253392459.png)

### but we want
![image](https://user-images.githubusercontent.com/5485798/45371202-ed66b380-b5e1-11e8-9a30-1c65efeea629.png)

that's because the field value is markdown-generated HTML, and here is a list within a list. we want the sub-list to have the borderline thing, but not the parent list.

but good tip - I can use `panel-vertical-narrow` elsewhere:)

Reply:
is that adding the class to the `ul`? couldn't you add it to the `li`s instead? or does that give you some weird alignment issues?